sdk/communication/azure-communication-chat/pom.xml

```diff
@@ -50,12 +50,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.0.4</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
+      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
```

Comment:
For network traversal this is 1.0.4. Shouldn't this also be 1.0.4 here? @g2vinay

Reply:
yes, we should update the tag to dependency here, if its supposed to depend on last released version.